Makefile

```diff
@@ -7,20 +7,19 @@ export GO15VENDOREXPERIMENT=1
 VERSION := 0.0.1-$(shell date "+%Y%m%d%H%M%S")
 LDFLAGS := "-s -X main.version=${VERSION}"
 BINDIR := ./rootfs/bin
-DEV_REGISTRY ?= $(eval docker-machine ip deis):5000
+DEV_REGISTRY ?= $(docker-machine ip deis):5000
```

Comment:
As long as docker-machine exists in your PATH this should work (and we don't need any shell expansion). I worked with @arschles on this fix as it wasn't working as written with the eval before.